### DISCUSSION TITLE
New community call?

### DISCUSSION THREAD

**Discussion by MEK3DK** [microsoft/microsoft-ui-xaml#9181] [General]
I know the community call in September was the last in that format but is the new format going to happen or?

Have I missed it on some platform?

**Comment by MEK3DK** [comment]
Silence...
I must have missed something because there is no way that Microsoft could be this inactive in regards of WinUI 3.

**Comment by danielgruethling** [comment]
I don't think we missed it. I think it did just not happen. There is no announcement post collecting questions and it's not on their youtube channel either.

The activity around this repo has been declining a lot during the last year. Seems it's not getting better any time soon.

**Reply by ChaimCL** [reply]
Previously they said adjustments to be made quarterly, I had previously thought it might be in January, and version 1.5 looks to be in the pipeline, so maybe they are busy in the middle of this matter. I've also seen a lot of PRs in the CsWinRT program that have AOT in them.

**Reply by danielgruethling** [reply]
Quarterly starting from the last one in september would be in december. Then again on March, June, September. Granting there's holidays so January would probably be targeted and following community calls shift a bit but we have not heard anything not even an announcement for this month or for February.

Having a new version in the pipeline would be prime time to have this call and get feedback on it since the experimental version is already out.

If their new schedule for community calls can not be fulfilled either what are they going to do? Switch to a 6 month cycle?

**Reply by MEK3DK** [reply]
They moved from 1 month between the calls to 3 months, but they are still not able to set aside 5 minutes for somebody to make a post that says "Calls postponed/cancelled". And now apparently they are just silent.
So busy or not they have 0 resources that focus on their "customers".

**Reply by pjmlp** [reply]
I moved back to Web and distributed computing, when it was clear WinUI was yet another WinRT reboot since Windows 8 was introduced, following upon the Windows 8.1 UAP rewrites, Windows 10 UWP rewrites,  C++/CX forced rewrites due to C++/WinRT replacement with Visual C++ 6.0 like stone age tooling, .NET Native AOT lagging behid .NET Native, Project Reunion being announced, XAML Islands left behind for a while after being pushed on as the way forward Win32/UWP, yet another reboot with WinUI 3.0/WinAppSDK.

Really, WinUI is now only for Windows team themselves. It wouldn't surprise me if MAUI goes Blazor or  WPF on Windows.

**Reply by MEK3DK** [reply]
And when XAML Island was presented in WinUI they only had a basic C++ example/implementation... In WinUI it is like everything has to be extra complicated to do, no simple properties or helper classes. But again that is probably because the resources are so limited.

Yes WPF in MAUI would make sense. The WPF team is at least doing something and a few improvements to WPF would be great.

**Comment by duncanmacmichael** [comment]
Sorry we've been quiet. We've been working on a more sustainable plan to engage with the developer community long term, and we're excited to share the plan with you all soon!

**Reply by dicksonkimeu** [reply]
Oh, nice !
